### PR TITLE
chore: extra neo env config

### PR DESCRIPTION
**Pull request by kaladinlight**
## Description

Set up DNS for neo environment and adding a couple more config updates. Also removed `cafe` env.

https://neo.shapeshift.com/#/connect-wallet?returnUrl=/trade

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [x] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [ ] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

N/A

> What protocols, transaction types or contract interactions might be affected by this PR?

None

## Testing

- Checkout and push to `neo` branch and see

### Engineering

:point_up:

### Operations

N/A

### CODE REVIEW

File: .github/workflows/cloudflare.yml
Comment (NeOMakinG) [inline .github/workflows/cloudflare.yml:18]: Seems like neo is present 2 times, I don't get why 😆

Comment (kaladinlight) [reply .github/workflows/cloudflare.yml]: good catch

**Review by gomesalexandre — COMMENTED** [review]
LGTM assuming changes in neo are propagated upstream, waiting for CI

**Review by NeOMakinG — APPROVED** [review]

**Pull request merged**
